### DISCUSSION TITLE
Publish lint rules

### DISCUSSION THREAD

**Discussion by vanniktech** [slackhq/slack-lints#6] [General]
Would be nice if it were distributed, I'd just add it onto my project, see which rules I like

Now digging through sources / tests is cumbersome

**Comment by ZacSweers** [comment]
We're unlikely to publish all of these as-is (many are highly opinionated or specific to our codebase), but I'm open to requests for specific lints!

**Comment by chris-mitchell** [comment]
It looks like the `DoNotMock` lint rules at https://github.com/slackhq/slack-lints/tree/main/slack-lint/src/main/java/slack/lint/mocking are fairly generic and would benefit our projects.  It'd be great to have those published in some form.  Thank you!
